pom.xml

```diff
@@ -220,6 +220,13 @@
 	      <groupId>org.apache.maven.plugins</groupId>
 	      <artifactId>maven-jar-plugin</artifactId>
 	      <version>3.3.0</version>
+          <configuration>
+            <archive>
+              <manifestEntries>
+                <Automatic-Module-Name>org.spdx.library</Automatic-Module-Name>
+              </manifestEntries>
+            </archive>
+          </configuration>
 		</plugin>
```

Comment:
Just a nit - the indentation is a bit off.  The POM file unfortunately has a mix of tabs and spaces - you can just copy/paste the white space from preceding the `<version>` to make it consistent.

Reply:
The indentation is way off across the entire file, not just a mix of tabs and spaces but also incorrect indentation levels (within single elements) and a mix of 2, 4 and 8 space indentation. Wouldn't it make sense to just reformat the entire file? Whatever I change about this new block will not make the indentation consistent.

Reply:
Yes - it's a mess.

I'll go ahead and merge it then run it through a pretty printer to reformat.